### ISSUE TITLE
Need to update aiohttp - current version has a low severity vulnerability

### ISSUE THREAD

**Issue by mffiedler** [openshift/svt#681]
See https://github.com/openshift/svt/security/dependabot/reliability/requirements.txt/aiohttp/open

Need to upgrade to 3.7.4 or higher for reliability

**Comment by qiliRedHat** [comment]
@mffiedler I can't open the like you provided. When did you get this security alert?
We already have `aiohttp==3.7.4.post0` in last merge for cerberus integration on Jul 14, 2021.
https://github.com/openshift/svt/blob/4002073f006c967d069a10550fe2e8787b057f42/reliability/requirements.txt#L3
`aiohttp==3.7.4.post0` is the latest version from https://docs.aiohttp.org/en/stable/ 
>Current version is 3.7.4.post0.

**Comment by mffiedler** [comment]
Here's copy/paste of the text.   I'll bet the "post0" is throwing off their detection tool.  Possibly change the version to >=3.7.4.    But this is not high priority.   I only noticed because it was an automated GitHub notification.

1 aiohttp vulnerability found in reliability/requirements.txt 7 days ago
Remediation
Upgrade aiohttp to version 3.7.4 or later. For example:

aiohttp>=3.7.4
Always verify the validity and compatibility of suggestions with your codebase.

Details
GHSA-v6wp-4m6f-gcjg
low severity
Vulnerable versions: < 3.7.4
Patched version: 3.7.4
Impact
What kind of vulnerability is it? Who is impacted?

Open redirect vulnerability — a maliciously crafted link to an aiohttp-based web-server could redirect the browser to a different website.

It is caused by a bug in the aiohttp.web_middlewares.normalize_path_middleware middleware.

Patches
Has the problem been patched? What versions should users upgrade to?

This security problem has been fixed in v3.7.4. Upgrade your dependency as follows:
pip install aiohttp >= 3.7.4

Workarounds
Is there a way for users to fix or remediate the vulnerability without upgrading?

If upgrading is not an option for you, a workaround can be to avoid using aiohttp.web_middlewares.normalize_path_middleware in your applications.

References
Are there any links users can visit to find out more?

aiohttp @ PyPI
GHSA-v6wp-4m6f-gcjg
OWASP page on open redirects
For more information
If you have any questions or comments about this advisory:

Open an issue in the aiohttp repo
Email us at wk+aio-libs-security@sydorenko.org.ua and/or andrew.svetlov+aio-libs-security@gmail.com
Credit: Jelmer Vernooĳ and Beast Glatisant.
